### PR TITLE
ApiV4: PUT /teams/{team_id}

### PR DESCRIPTION
**Pull request by andreistanciu24**

### CODE REVIEW

File: source/v4/teams.yaml [hunk 1]
```diff
@@ -101,6 +101,60 @@
...
+          name: body
+          description: Team object that is to be updated
+          required: true
+          schema:
+            type: object
+            required:
+              - id
+              - display_name
+            properties:
+              id:
```
Comment (jwilander) [inline source/v4/teams.yaml]: Can you remove `id` from here and from required?

File: source/v4/teams.yaml [hunk 2]
```diff
@@ -101,6 +101,60 @@
...
+                type: string
+              display_name:
+                type: string
+              description:
+                type: string
+              email:
+                 type: string
```
Comment (jwilander) [inline source/v4/teams.yaml]: Email isn't updatable, please remove

File: source/v4/teams.yaml [hunk 3]
```diff
@@ -101,6 +101,60 @@
...
+              type:
+                type: string
```
Comment (jwilander) [inline source/v4/teams.yaml]: Type isn't updatable, please remove

File: source/v4/teams.yaml [hunk 4]
```diff
@@ -101,6 +101,60 @@
...
+          type: string
+        - in: body
+          name: body
+          description: Team object that is to be updated
+          required: true
+          schema:
+            type: object
+            required:
+              - id
+              - display_name
```
Comment (jwilander) [inline source/v4/teams.yaml]: Can you mark all fields that don't get removed as required?

**Comment by andreistanciu24** [comment]
done

**Pull request closed without merging**